test/common_test/worlds/test.world

```diff
@@ -379,5 +379,130 @@
         <child>link5</child>
       </joint>
     </model>
+
+    <model name="pendulum_with_base">
```

Comment:
I think adding a model here caused a test expectation to file in an different unrelated test, see https://github.com/gazebosim/gz-physics/actions/runs/14040188656/job/39311988868?pr=723

```
  /github/workspace/tpe/plugin/src/SDFFeatures_TEST.cc:98: Failure
  Expected equality of these values:
    7u
      Which is: 7
    tpeWorld->GetChildCount()
      Which is: 8
```

We can just fix it by changing the test expectation in that test.